packages/webapp/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  webapp:
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - /app/node_modules # prevents "node_modules" from getting overwritten
```

Comment:
I think we kind of want node modules to be overridden locally while we are actively devving or else it will get stale